### PR TITLE
Run tests on Github actions.

### PR DESCRIPTION
**Pull request by halilb**
Trying to run it.

### CODE REVIEW

**Review by chrisdavies — APPROVED** [review]
LGTM. Thanks for doing this.

File: package.json
Comment (halilb) [inline package.json:15]: `prepare` was running on a fresh `npm install`. So changed it with `prepack` to prevent running the tests twice on the cli.

File: package.json
Comment (halilb) [inline package.json:36]: `onchange` fails the `npm install` on Windows because of `fsevents`. Marking it as optional solves the problem.

**Comment by halilb** [comment]
> LGTM. Thanks for doing this.

Could you re-review this @chrisdavies? Made some changes to make it work on three operating systems.

File: src/integration-test/minidoc-integration.test.ts
Comment (chrisdavies) [inline src/integration-test/minidoc-integration.test.ts:828]: I say we disable Windows Webkit for now, instead of disabling this test. What do you think?

**Review by chrisdavies — APPROVED** [review]
LGTM w/ possible tweak.

File: src/integration-test/minidoc-integration.test.ts
Comment (halilb) [reply src/integration-test/minidoc-integration.test.ts]: Makes sense! While trying to exclude webkit tests on Windows, they somehow succeeded :) Not sure what was wrong yesterday.

Screen Shot 2021-03-09 at 14 38 45

**Comment by halilb** [comment]
> LGTM w/ possible tweak.

Could you give me contributor access to the repository? I can't merge this PR right now.

**Comment by halilb** [comment]
Pinging to merge this @chrisdavies.

**Comment by chrisdavies** [comment]
Sorry, was heads down yesterday! You should be good to go. I'll let you merge it as a way to test that I set your access up correctly.

**Comment by halilb** [comment]
> Sorry, was heads down yesterday! You should be good to go. I'll let you merge it as a way to test that I set your access up correctly.

Thank you. It worked!

**Pull request merged**
